modules/apps/frontend-js/frontend-js-react-web/src/main/resources/META-INF/resources/js/hooks/useStateSafe.es.js

```diff
@@ -12,9 +12,10 @@
  * details.
  */
 
-import {useIsMounted} from 'frontend-js-react-web';
 import React from 'react';
 
+import useIsMounted from './useIsMounted.es';
```

Comment:
Are we happy with this kind of modifications? @jbalsas @wincent ?

I don't know why we were using the package name instead of the local route. Was that on purpose? :thinking: 

----

Apparently the file was moved from other project and it wasn't updated :point_right: https://github.com/liferay/liferay-portal/commit/9bdf665602d8292628273ee3b4a4351ea37fd870

Reply:
Yeah, I'm okay with this :)

Reply:
Legit.